dtbase/tests/test_api_sensors.py

```diff
@@ -54,3 +54,129 @@ def test_list_sensors_of_a_type(client):
         response = client.get("/sensor/list/weather")
         assert response.status_code == 200
         assert isinstance(response.json, list)
+
+
+@pytest.mark.skipif(not DOCKER_RUNNING, reason="requires docker")
+def test_insert_sensor_readings(client):
+    with client:
+        # Insert a sensor type and a sensor
+        response = insert_weather_type(client)
+        assert response.status_code == 201
+        sensor = {
```

Comment:
Optional, but since many tests need to create the same sensor type and sensor to get started, maybe that could be put in a function that can be called by all of the tests that need it? `test_sensors.py` and `test_locations.py` do something similar.

Reply:
Also good idea. Made a `insert_weather_sensor` function.